plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -8,19 +8,19 @@ module Ide.Plugin.Fourmolu (
     provider,
 ) where
 
-import           Control.Exception           (try)
-import           Control.Lens                ((^.))
+import           Control.Exception               (try)
+import           Control.Lens                    ((^.))
 import           Control.Monad.IO.Class
-import           Data.Bifunctor              (first)
-import qualified Data.Text                   as T
-import           Development.IDE             hiding (pluginHandlers)
-import           Development.IDE.GHC.Compat  as Compat hiding (Cpp)
+import           Data.Bifunctor                  (first)
+import qualified Data.Text                       as T
+import           Development.IDE                 hiding (pluginHandlers)
+import           Development.IDE.GHC.Compat      as Compat hiding (Cpp)
 import qualified Development.IDE.GHC.Compat.Util as S
```

Comment:
I don't know how this file was passing CI before? Seeing as this line causes `stylish-haskell` to reformat all the other imports (I personally detest this behaviour, but that's an argument for another day).